### PR TITLE
Add 10s rewind button to controlbar for VOD/DVR playback

### PR DESCRIPTION
**Pull request by egreaves**
### Changes proposed in this pull request:
Add a 10s rewind button to the controlbar, visible only during VOD/DVR playback. 

Fixes #
JW7-2515

### CODE REVIEW

**Comment by hghazzi** [comment]
test this please.

File: assets/fonts/svg/jw7-icomoon-icon-setup.json [hunk 1]
```diff
@@ -1,613 +1,470 @@
...
+      "bgColor": 16777215,
+      "classSelector": ".icon"
+    },
+    "historySize": 100,
+    "showCodes": true,
+    "gridSize": 16,
+    "showGrid": true
+  },
+  "IcoMoonType": "icon-set"
 }
```
Comment (hghazzi) [inline assets/fonts/svg/jw7-icomoon-icon-setup.json]: Did anything actually change in this file?

File: src/css/imports/vars.less [hunk 2]
```diff
@@ -31,7 +31,7 @@
 @font-color: @active-color;
 
 // UI
-@ui-padding: 0.4em;
+@ui-padding: 0.5em;
```
Comment (hghazzi) [inline src/css/imports/vars.less]: Is this a universal change that was decided upon?

File: src/js/view/controlbar.js [hunk 3]
```diff
@@ -464,6 +470,19 @@ define([
...
+                duration = this._model.get('duration'),
+                rewindPosition = currentPosition - 10,
+                startPosition = 0;
+
+            // duration is negative in DVR mode
+            if (utils.adaptiveType(duration) === 'DVR') {
+                startPosition = duration;
+            }
+            // Seek 10s back. Seek value should be >= 0 in VOD mode and >= (negative) duration in DVR mode
+            this._api.seek(Math.max(rewindPosition, startPosition));
```
Comment (hghazzi) [inline src/js/view/controlbar.js]: It would be nice to have unit tests for this, if its quick and easy to setup a test file for controlbar.js

**Comment by johnBartos** [comment]
test this please

**Comment by johnBartos** [comment]
Going to merge this, if/when the new icon comes in we can re-PR it

**Pull request merged**